templates.js

```diff
@@ -291,6 +291,149 @@ formTimingTracking(function (eventData) {
         },    
       ],
     },
+    {
+      id: 'formAnalytics',
+      name: 'Form Analytics',
```

Comment:
Please provide a description for form analytics template. Users won't know how it's different from `Form timing tracking` template right above.